apps/passport-server/src/apis/zuconnect/zuconnectTripshaAPI.ts

```diff
@@ -62,14 +63,26 @@ export class ZuconnectTripshaAPI {
     const url = urljoin(this.baseUrl, "tickets", this.authKey);
     const fetchResult = await fetch(url);
     const data = await fetchResult.json();
-    const parsed = z
-      .object({ tickets: z.array(ZuconnectTripshaNormalizedNameSchema) })
-      .safeParse(data);
 
-    if (parsed.success) {
-      return parsed.data.tickets;
+    if (_.isArray(data.tickets)) {
```

Comment:
nit but it's a bit strange that you're using zod for `data.tickets`, but not `data` itself in favor of lodash

Reply:
I only want to validate that there is an array at all, without validating what its contents are (because if I validated it as an array of valid tickets, it might fail). I only want to apply validation to the members of the array, not to the array itself, and then ignore the individual items which fail validation.

Reply:
got it